### DISCUSSION TITLE
Transaction Amount Sign for Import function

### DISCUSSION THREAD

**Discussion by Samurai97** [TheAxelander/OpenBudgeteer#273] [Q&A] [answered]
Can someone explain to me whether a transaction should be signed +ve or -ve for an expense and the same question also for income?  I'm probably trying to use this tool incorrectly as I am attempting to build up the data by importing historic transactions from Bank imports.  Obviously bank systems hold negative and positive transactions but maybe oppoiste from what Budgeteer is expecting?

**Comment by TheAxelander** [comment, marked as answer]
Income values should be positive (e.g. 5), expenses should be negative (e.g. -5)

**Reply by Samurai97** [reply]
The import functionality appears to do the opposite. The import file has separate columns for debit and credit, which the importer supports, and all numbers are positive.

When I preview the data, expenses are shown as positives and income as negatives. i.e the inverse of your response.

...

Ahh .. don't worry, I've just switched the debit and credit mappings around and that then follows your guidance.
